include/aspect/material_model/utilities.h

```diff
@@ -268,6 +268,16 @@ namespace aspect
 
 
 
+      /**
+       * Used for calculating average elastic shear modulus and viscosity
```

Comment:
can you explain what it does?

Reply:
I was just moving this function, but after checking what it actually does, I found it was just duplicating code, so I removed it. That also solves your comment about the name below.